zero_bin/leader/src/cli.rs

```diff
@@ -51,23 +51,10 @@ pub(crate) enum Command {
         /// The previous proof output.
         #[arg(long, short = 'f', value_hint = ValueHint::FilePath)]
         previous_proof: Option<PathBuf>,
-        /// If provided, write the generated proofs to this directory instead of
-        /// stdout.
-        #[arg(long, short = 'o', value_hint = ValueHint::FilePath)]
-        proof_output_dir: Option<PathBuf>,
         /// Network block time in milliseconds. This value is used
         /// to determine the blockchain node polling interval.
         #[arg(short, long, env = "ZERO_BIN_BLOCK_TIME", default_value_t = 2000)]
```

Comment:
Do we want 2k here?

Reply:
Actually @atanmarko, your new `DEFAULT_BLOCK_TIME` constant is supposed to do the same as `ZERO_BIN_BLOCK_TIME` right? So we can probably reuse the latter as this is already configurable (cf my initial comment).

@muursh we probably want this configurable, but 2sec as default value seems fair given the initial networks we'd be supporting IMO

Reply:
@Nashtare `DEFAULT_BLOCK_TIME` is actually never used in `unwrap_or_default()` because CLI `ZERO_BIN_BLOCK_TIME` has default time. I'll remove  `DEFAULT_BLOCK_TIME`